tests/sqlparser_snowflake.rs

```diff
@@ -506,3 +507,230 @@ fn test_alter_table_swap_with() {
         _ => unreachable!(),
     };
 }
+
+#[test]
+fn test_drop_stage() {
+    match snowflake_and_generic().verified_stmt("DROP STAGE s1") {
+        Statement::Drop {
+            names, if_exists, ..
+        } => {
+            assert!(!if_exists);
+            assert_eq!("s1", names[0].to_string());
+        }
+        _ => unreachable!(),
+    };
+    match snowflake_and_generic().verified_stmt("DROP STAGE IF EXISTS s1") {
+        Statement::Drop {
+            names, if_exists, ..
+        } => {
+            assert!(if_exists);
+            assert_eq!("s1", names[0].to_string());
+        }
+        _ => unreachable!(),
+    };
+
+    snowflake_and_generic().one_statement_parses_to("DROP STAGE s1", "DROP STAGE s1");
+
+    snowflake_and_generic()
+        .one_statement_parses_to("DROP STAGE IF EXISTS s1", "DROP STAGE IF EXISTS s1");
+}
+
+#[test]
+fn test_create_stage() {
+    let sql = "CREATE STAGE s1.s2";
+    match snowflake().verified_stmt(sql) {
+        Statement::CreateStage {
+            or_replace,
+            temporary,
+            if_not_exists,
+            name,
+            comment,
+            ..
+        } => {
+            assert!(!or_replace);
+            assert!(!temporary);
+            assert!(!if_not_exists);
+            assert_eq!("s1.s2", name.to_string());
+            assert!(comment.is_none());
+        }
+        _ => unreachable!(),
+    };
+    assert_eq!(snowflake().verified_stmt(sql).to_string(), sql);
+
+    let extended_sql = concat!(
```

Comment:
can you add at least one test case that roundtrips `CREATE [OR REPLACE] [TEMPORARY] STAGE` (via `one_statement_parses_to`)?

Reply:
I am testing roundtrip with:
```
assert_eq!(snowflake().verified_stmt(sql).to_string(), sql);
```
In case it's not sufficient, please provide more details (or example) to help me understand what did you mean. `one_statement_parses_to` helps to verify if similar SQLs are semantically the same and I couldn't see how to fit this here.